src/mono/sample/wasm/browser-nextjs/package-lock.json

```diff
@@ -0,0 +1,7418 @@
+{
+  "name": "browser-nextjs",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
+      "dependencies": {
+        "dotnet": "file:../../../../../artifacts/bin/native/net7.0-Browser-Debug-wasm/",
```

Comment:
Is there a way how to use configuration dependent `package-lock.json` ? In the build usually created Release configuration. Does that means that 2 builds for Debug and for Release needed?

Reply:
This is in-tree sample of the runtime repo and so it's using npm package as it was built in the `artifacts` folder.
I updated the sample to `npm install` the `@microsoft/dotnet-runtime` from MSBuild/csproj during the build of the sample, so that we would have the same Debug/Release configuration as the C# part. 

The real solution would be to start publishing `@microsoft/dotnet-runtime` npm package into public repository. I don't know if or when we would get there.